src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -62,7 +62,18 @@ By default, `<script>` tags are processed by Astro.
 </script>
 ```
 
-To avoid bundling the script, you can use the `is:inline` directive.
+#### All processed scripts become module scripts
+
+This has several benefits:
+- Rendering is not blocked. The browser continues to process the rest of the html while the module script and its dependencies load.
+- The browser waits for html to be processed before executing module scripts. You do not need to listen for the "load" event.
+- `async` and `defer` attributes become unnecessary. Module scripts are always deferred.
+
+The `async` attribute is valuable for normal scripts as it prevents them from blocking rendering. However, module scripts already have this behavior and adding this attribute will make module scripts execute before the page has fully loaded. This is probably not what you want.
```

Comment:
```suggestion
:::note
The `async` attribute is valuable for normal scripts because it prevents them from blocking rendering. However, module scripts already have this behavior. Adding `async` to a module script will cause it to execute before the page has fully loaded. This is probably not what you want.
:::
```
Now, here's a case where I think this makes a great "note"! (Or "tip" even?)

Reply:
I agree, and I think note is still better - users do this hoping to delay things but they would be surprised to know it has the opposite effect.